cmd/restore.go

```diff
@@ -47,6 +47,7 @@ import (
 	"time"
 )
 
+//Restore struct
```

Comment:
Adding empty definitions merely to satisfy the linter defeats the purpose of the linter. Let's remove these or replace them w idiomatic go docs. https://blog.golang.org/godoc-documenting-go-code